hack/generator/pkg/astmodel/enum_definition.go

```diff
@@ -104,3 +110,14 @@ func (enum *EnumDefinition) createValueDeclaration(value EnumValue) ast.Spec {
 
 	return valueSpec
 }
+
+// FindRelatedDefinitions returns a set of definitions related to this one
```

Comment:
I am not seeing anywhere that this method is actually called...?

Reply:
Also since `EnumDefinition` implements `Definition`, and `Definition` `HasRelatedDefinitions`, shouldn't enum have a `CreateRelatedDefinitions`? I'm not seeing it.

Reply:
Oh, but `DefinitionName` is embedded and `DefinitionName` has an implementation of `CreateRelatedDefinitions` already, which we are getting because of struct embedding...?

But do we _really want_ that definition? It feels like we don't, we want our own definition here that farms out to `baseType` like `FindRelatedDefinitions` does below?

Which is maybe a long-winded way of saying, do we really want to embed `DefinitionName` here, or do we just want a property of type `DefinitionName`, but not embedded?

Reply:
Ooo, good catch - this is debris from when I tried something else; the method is needed, but it has the wrong name, wrong type, and wrong implementation. 😜 

I think I'll just replace it with the right thing.

Reply:
Done.